src/server/db_slice.cc

```diff
@@ -400,15 +400,28 @@ DbSlice::AutoUpdater::AutoUpdater(const Fields& fields) : fields_(fields) {
 
 DbSlice::AddOrFindResult& DbSlice::AddOrFindResult::operator=(ItAndUpdater&& o) {
   it = o.it;
-  exp_it = ExpireIterator{};  // ItAndUpdater doesn't have exp_it
+  exp_it = o.exp_it;
   is_new = false;
   post_updater = std::move(o).post_updater;
   return *this;
 }
 
+DbSlice::ItAndUpdater DbSlice::FindMutable(const Context& cntx, string_view key) {
```

Comment:
I suggest instead of duplicating the code with the other find mutable that checks the type you can use OBJ_TYPE_MAX to indicate inside find mutable that you will not check the type, and you can have this set by default so the calls to this function which dont care about obj type will send only cntx and key

Reply:
This will be a big change, as the `type`-accepting versions return an `optional`, so it means that every call to previous `FindExt()` will now have to check `.value()`, which is not always desired..